public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
jeżeli chodzi o grafiki to lepiej gdybyś wykorzystała icon-fonts albo sprit'a. Tak teraz obsługujemy na stronach takie małe assety. Nie chcemy wczytywać wiele małych plików a jeden zawierający wiele